### PR TITLE
Fix & consolidate gulp-codecov entry

### PR DESCRIPTION
**Pull request by brody2consult**
Alternative to #22: consolidate and fix gulp-codecov entry but do not disable it.

Only wanted if my contribution in <https://github.com/eddiemoore/gulp-codecov/pull/62> is accepted.

### CODE REVIEW

**Comment by brody2consult** [comment]
~~or if eddiemoore/gulp-codecov#69 is integrated~~

**Comment by eddiemoore** [comment]
gulp-codecov fixes published @3.0.5

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "standard-packages",
   "description": "List of Packages that Use JavaScript Standard Style",
-  "version": "3.4.5",
+  "version": "3.4.6-dev",
```
Comment (LinusU) [inline package.json]: Could you remove the commit that bumps this version number?

Comment (brody2consult) [reply package.json]: > Could you remove the commit that bumps this version number?

now gone

**Review by LinusU — APPROVED** [review]
Thanks!

**Pull request merged**